CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
 
 ### Removed
 
+- Individual etcd member's services were removed. Use hostname and subdomain of etcd pod instead.
```

Comment:
Yes I meant, `http://<pod-hostname>.<pod-subdomain>.<namespace>.svc.cluster.local`. Okay I must have missed that.